css/css-font-loading/idlharness.https.html

```diff
@@ -14,10 +14,15 @@
   idl_array => {
     idl_array.add_objects({
       Document: ['document'],
-      FontFace: ['new FontFace("family", "src")'],
-      FontFaceSetLoadEvent: ['new FontFaceSetLoadEvent("type")'],
+      FontFace: ['fontFace'],
+      FontFaceSetLoadEvent: ['fontFaceSetLoadEvent'],
       FontFaceSet: ['document.fonts'],
     });
+    self.fontFace = new FontFace("family", "src");
+    // The `fontFace.loaded` promise will be rejected, so handle that to
+    // avoid an unhandled promise rejection manifesting as a harness error.
+    self.fontFace.loaded.catch(() => {});
+    self.fontFaceSetLoadEvent = new FontFaceSetLoadEvent("type");
```

Comment:
Why move this one?

Reply:
Just for consistency, since the variable names are what will show up in the test names.

It does mean that the order matters, but I guessed that no implementation would support `FontFaceSetLoadEvent` without first supporting `FontFace`, so if the above throws it wouldn't result in less coverage in practice.